wasm/validator/src/permission.rs

```diff
@@ -48,6 +51,8 @@ pub enum PermissionTokenConversionError {
     // TODO: Improve this error
     /// Unexpected parameter value.
     Value(alloc::string::String),
```

Comment:
Do we need `Value` and `Param` variants anymore?

Reply:
You're right, we don't